lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Utils.pm

```diff
@@ -261,7 +261,7 @@ sub serialize_relationships {
             join("\t",
                  $_->link->type->name,
                  (sprintf "%09d", $_->link_order // 0),
-                 $_->target_key)
+                 $_->target->id)
```

Comment:
That doesn’t seem to be equivalent to the previous `target_key`.

Reply:
Yeah, that's true, though the order is mostly arbitrary to ensure it's stable, so I picked the smallest working key.

Actually I'm not sure this code is needed after b90e1e7f3c1d96a7882196f734a857d7fe718fc3, we can probably just return them in the order they're loaded.

Reply:
There's also an old issue where XML element attributes are serialized in random order, so `Controller::Role::ETags` isn't very effective and makes most of the sorting we do for naught. I haven't checked if the `XML::LibXML` switch fixes that.